gcloud/datastore/test_connection.py

```diff
@@ -520,7 +520,7 @@ def test_run_query_wo_eventual_w_transaction(self):
         DATASET_ID = 'DATASET'
         KIND = 'Nonesuch'
         CURSOR = b'\x00'
-        TRANSACTION = 'TRANSACTION'
+        TRANSACTION = b'TRANSACTION'
```

Comment:
`TRANSACTION` gets copied into a protobuf `bytes_field`, and hence must be bytes.  `DATASET_ID` is passed into the bits which generate the API URL, and therefore doesn't need to be bytes.